dockerfiles/Dockerfile.android-publisher

```diff
@@ -0,0 +1,7 @@
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/android:latest as android
+
+WORKDIR /app
+
+COPY . .
```

Comment:
Would this copy _everything_ in the working directory? This is avoided in other parts of the pipeline as it could add unnecessary files such as build directories.